guides/v2.1/frontend-dev-guide/themes/theme-structure.md

```diff
@@ -77,13 +77,13 @@ The directories and files structure described below is the most extended one. It
     </tr>
     <tr>
       <td colspan="1">
-        <code>/&lt;Vendor&gt;_&lt;Module&gt;/web/css/source</code>
+        <code>/&lt;Vendor&gt;_&lt;Module&gt;/web/css/</code>
       </td>
       <td colspan="1">
         optional
       </td>
       <td colspan="1">
-          Module-specific styles (<code>.css</code> and/or <code>.less</code> files). General styles for the module are in the <code>_module.less</code> file, and styles for widgets are in <code>_widgets.less</code>.
+          Module-specific styles (<code>.css</code> and/or <code>.less</code> files). General styles for the module are in the <code>_module.less</code> file, and styles for widgets are in <code>_widgets.less</code><br/><br/>Ex: [Module_Theme]({{ site.mage2300url }}app/code/Magento/Theme/view/frontend/web/css) or description field should be corrected</code>.
```

Comment:
The `guides/v2.1/frontend-dev-guide/themes/theme-structure.md` file is symlinked to the v2.2 and v2.3 files, so if you really want to add a link to the magento code repo here using the `{{ site.mage2300url }}` variable, you must remove the symlinks, create new files for 2.2 and 2.3, and add the version-specific variable to each file just like #3423.

Reply:
@gelanivishal, you still need to create unique files for 2.2 and 2.3 if you want to use the `{{site.mage2100url}}` var.

Reply:
@gelanivishal, if we publish this as-is, the output will create 2 different versions of the topic (2.1, 2.2, and 2.3), but all three will link to the 2.3 branch of magento/magento2. Each version of the docs should link to the correct version of the code repo, that's why I'm asking you to delete symlinks and create new files if you insist on including this new version-specific URL.